models/classes/class.QtiTestConverter.php

```diff
@@ -162,7 +162,7 @@ private function componentToArray(QtiComponent $component)
                         $array[$property->getName()][] = $item;
                     }
                 } else {
-                    $array[$property->getName()] = is_string($value) ? html_entity_decode($value) : $value;
+                    $array[$property->getName()] = $value;
```

Comment:
Can you please just confirm that tests are still passing? Thanks!

Reply:
@gabrielfs7 no change in the result of the unit tests